community.md

```diff
@@ -141,8 +141,7 @@ This is a calendar of regular online events.  It might not be exhaustive.
           style="border:solid 1px #777"
           width="800"
           height="600"
-          frameborder="0"
-          scrolling="no"></iframe>
+          style="border: 0; scrolling: no;"></iframe>
```

Comment:
according to the error message shouldn't we also remove scrolling?

Reply:
Yes, this should be replaced by `overflow: hidden`. Also this introduces two `style` attributes - these should be merged.

Reply:
Those bugs have been resolved. Thanks for the pointers.